koku/masu/external/kafka_msg_handler.py

```diff
@@ -629,6 +685,9 @@ async def process_messages(msg, loop=EVENT_LOOP):
             async_id = await loop.run_in_executor(pool, summarize_manifest, report_meta)
```

Comment:
Think we need to include the `report_id` here.

Reply:
Fixed